README.md

```diff
@@ -1,2 +1,201 @@
+selected authenticator. The API endpoints specify a set of error 
+scenarios which can be found at WSO2 official documentation for this connector.
+
+```agsl
+curl --location 'https://localhost:9443/api/identity/authn/v1/init-authenticator' \
+--header 'accept: application/json' \
+--header 'Content-Type: application/json' \
+--data '{
+  "flowId":"c9e2bed6-839c-4754-a2a1-5f4fac312055",
+  "authenticator":"EmailOTP"
+}'
+```
+
+## Steps to configure
+
+Stop the WSO2 Identity Server if running.
+
+1) Extract the zip file and you may be able to find below artifacts.
+   * `org.wso2.carbon.identity.oauth2.grant.rest.framework-x.x.x.jar`
+   * `api#identity#authn#v1.war`
+   * `identity.oauth2.grant.auth.rest.handler-1.0.0.jar`
+   * `dbscrips` directory
+   * `rest-auth.properties`
+
+2) Copy `org.wso2.carbon.identity.oauth2.grant.rest.framework-x.x.x.jar` file into the 
+`<IS-Home>/repository/components/dropins` directory.
+
+3) Copy `api#identity#authn#v1.war`
+file into the `<IS-Home>/repository/deployment/server` directory.
+
+4) Copy `identity.oauth2.grant.auth.rest.handler-x.x.x.jar` file into the
+`<IS-Home>/repository/components/libs` directory.
+
+5) Get the `org.wso2.carbon.extension.identity.smsotp.common-x.x.x.jar` and 
+`org.wso2.carbon.extension.identity.emailotp.common-x.x.x.jar` and copy in to 
+the `<IS-Home>/repository/components/dropins` directory. 
+(The 02 JARs are required to this connector in order to send the OTP for 
+respective channel if [EmailOTP](https://github.com/wso2-extensions/identity-outbound-auth-sms-otp/tree/master/component/common) 
+or [SMSOTP](https://github.com/wso2-extensions/identity-outbound-auth-email-otp/tree/master/component/common) 
+would enable in an authentication steps)
+
+6) Execute the [db-scripts](https://github.com/DInuwan97/identity-oauth2-grant-mfa/tree/dev-extenssion-features/components/org.wso2.carbon.identity.oauth2.grant.rest.framework/src/main/resources/dbscripts) 
+in the WSO2 Identity DB.
+
+7) Copy `rest-auth.properties` file into the 
+`<IS-Home>/repository/conf` directory. 
+(You can configure this property file configurations as your requirements. 
+Customize error code,error message and description can be attached here. 
+By default the file includes the default error messages.)
+
+
+### deployment.toml configurations
+
+#### Endpoint configurations
+
+```toml
+[[resource.access_control]]
+default_access = "allow"
+context = "(.*)/api/identity/authn/v1/authenticate"
+secure = "false"
+http_method = "POST"
+
+[[resource.access_control]]
+default_access = "allow"
+context = "(.*)/api/identity/authn/v1/init-authenticator"
+secure = "false"
+http_method = "POST"
+
+[[resource.access_control]]
+default_access = "allow"
+context = "(.*)/api/identity/authn/v1/auth-steps"
+secure = "false"
+http_method = "GET"
+
+[tenant_context.rewrite]
+custom_webapps=["/api/identity/authn/v1/"]
+```
+
+#### SMSOTP and EmailOTP Configurations
```

Comment:
are these configs specific for this connector? if not add a link to the official docs of the configs